nimble/syscfg.yml

```diff
@@ -77,6 +77,19 @@ syscfg.defs:
             integer for easy comparison.
         range: 50, 51, 52
         value: 50
+    BLE_ISO:
```

Comment:
I think we should have BLE_LL_FEAT_ISO (or similar) for controller configuration that defaults to BLE_ISO (which should set both host and controller support). This should keep this similar to other LL config options